app/models/repo.js

```diff
@@ -2,32 +2,32 @@ import DS from 'ember-data';
 
 function kCount(count){
   if(parseInt(count,10) > 999) {
-    return `${Math.round(count * 0.01) * 0.1}K`;
+    return `${Math.round((count * 0.01) * 0.1).toFixed(0)}K`;
   } else {
     return `${count}`;
   }
 }
 
 export default DS.Model.extend({
   name: DS.attr(),
-  full_name: DS.attr(),
+  fullName: DS.attr(),
   description: DS.attr(),
-  created_at: DS.attr(),
-  updated_at: DS.attr(),
-  pushed_at: DS.attr(),
-  stargazers_count: DS.attr(),
-  watchers_count: DS.attr(),
-  forks_count: DS.attr(),
+  createdAt: DS.attr(),
+  updatedAt: DS.attr(),
+  pushedAt: DS.attr(),
+  stargazersCount: DS.attr(),
+  watchersCount: DS.attr(),
+  forksCount: DS.attr(),
   language: DS.attr(),
 
-  star_count: Ember.computed('stargazers_count', function() {
-    return kCount(this.get('stargazers_count'));
+  starCount: Ember.computed('stargazersCount', function() {
```

Comment:
__[jshint]__
*  'Ember' is not defined.


<!-- Comment by [SideCI](https://sideci.com) -->